regnety/train.py

```diff
@@ -71,60 +67,61 @@
     weight_decay=5e-5,
     momentum=0.9,
     lr_schedule="half_cos",
-    log_dir="gs://ak-europe-train/logs",
-    model_dir="gs://ak-europe-train/models",
+    log_dir=log_location + "/logs",
+    model_dir=log_location + "/models",
 )
 
 
 train_prep_cfg = get_preprocessing_config(
-    tfrecs_filepath=train_tfrecs_filepath, augment_fn="default", mixup=True
+    tfrecs_filepath=train_tfrecs_filepath,
+    augment_fn="default",
+    mixup=False
 )
 
 val_prep_cfg = get_preprocessing_config(
-    tfrecs_filepath=val_tfrecs_filepath, augment_fn="val", mixup=False
+    tfrecs_filepath=val_tfrecs_filepath,
+    augment_fn="val",
+    mixup=False
 )
 
-
 logging.info(f"Training options detected: {train_cfg}")
 logging.info("Preprocessing options detected.")
 logging.info(
-    f"Training on TFRecords: {train_prep_cfg.tfrecs_filepath[0]} to {train_prep_cfg.tfrecs_filepath[-1]}"
-)
+    f"Training on TFRecords: {train_prep_cfg.tfrecs_filepath[0]} to {train_prep_cfg.tfrecs_filepath[-1]}")
 logging.info(
-    f"Validating on TFRecords: {val_prep_cfg.tfrecs_filepath[0]} to {val_prep_cfg.tfrecs_filepath[-1]}"
-)
+    f"Validating on TFRecords: {val_prep_cfg.tfrecs_filepath[0]} to {val_prep_cfg.tfrecs_filepath[-1]}")
 
 with strategy.scope():
     model = tutil.make_model(flops, train_cfg)
+    # model.load_weights(log_location + "/init_weights/" + flops.upper())
```

Comment:
Delete?

Reply:
I'm not sure if we should remove this or not. We used to load random weights stored in a bucket. Should that be reflected in the final implementation?

/cc @sayakpaul

Reply:
If there are changes in here that were not reflected in the trained models, it would be good to create and tag a branch with the code actually used to train version 1 on TF Hub.

That way you can delete any no-longer used code.

Reply:
Got it. In that case, I'll just uncomment this code. It was used in training.